file-formats/evto/type/zif_aff_evto_v1.intf.abap

```diff
@@ -0,0 +1,49 @@
+INTERFACE zif_aff_evto_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">General Information</p>
+    "! General information
+    BEGIN OF ty_general_information,
+      "! <p class="shorttext">SAP Object Type</p>
+      "! SAP Object Type
+      "! $required
+      sap_object_type TYPE c LENGTH 30,
+
+      "! <p class="shorttext">Major Version</p>
+      "! Major version
+      "! $minimum 1
```

Comment:
Just out of curiosity. Version 0.0.1 is not an allowed value?

Reply:
Exactly, we will default it with "1.0.0" and lower than that is not allowed.